static/src/stylesheets/module/content/_photo-essay.scss

```diff
@@ -0,0 +1,93 @@
+.content--article {
+    .element-image--photo-esssay {
+        padding-bottom: 0;
+        margin-bottom: 0.5rem;
+
+        &.fig--no-caption.fig--has-shares {
+            padding-bottom: 0px !important;
+        }
+
+        .block-share {
+            display: none !important;
+        }
+        .caption {
+            max-width: 100%;
+            color: $news-main-1;
+        }
+        .article__fullscreen {
+            background-color: transparent;
+            border:1px solid rgba($neutral-6,0.5);
+        }
+
+        &:hover .article__fullscreen {
+            background-color: white;
+            border: none;
+        }
+
+        .section-title {
+            &:before {
+                background-color: $news-main-2;
+                display: block;
+                content: '';
+                width: gs-span(2);
+                height: 0.5rem;
+                margin-bottom: calc(#{$gs-gutter} / 3);
+            }
+        }
+
+        &.element--supporting {
+          .caption {
+              max-width: 100%;
+          }
+        }
+
+        &.element--halfWidth {
+          width: 50%;
+          float: left;
+          margin-top: 0;
+          margin-bottom: 2px;
+
+          .inline-expand-image {
+            display: none;
+          }
+
+          &:before{
+            content: '';
+            display: block;
+            width: 2px;
+            top: 0px;
+            bottom: 0px;
+            z-index: 300;
+            position: absolute;
+            background-color: white;
+          }
+
+          & + p {
+            clear: both;
+            padding-top: 6px;
+          }
+
+        }
+
+    }
+
+
+    .element-pullquote--photoEssay {
```

Comment:
I think we should not use camelCase here. Would it be possible to rewrite it into `.element-pullquote--photo-essay`?